mmv1/description-copy.go

```diff
@@ -21,13 +21,262 @@ func CopyAllDescriptions() {
 		"attributes:",
 	}
 
-	for i, id := range identifiers {
-		CopyText(id, len(identifiers)-1 == i)
+	for _, id := range identifiers {
+		CopyText(id)
 	}
+
+	copyComments()
+}
+
+// TODO rewrite: ServicePerimeters.yaml is an exeption and needs manually fixing the comments over after switchover
```

Comment:
Will ServicePerimeters.yaml block future automatic refreshes or the comment just won't be copied over

Reply:
No, ServicePerimeters.yaml  will not block the automatic refreshes.

The comments of ServicePerimeters.yaml  are copied to the Go yaml files. The copied comments are not accurate and need to be fixed after the switch over.